<a href="https://colab.research.google.com/github/AngelTorresB/ProyectoEstadistica/blob/main/ProyectoEstadistica.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

<P ALIGN=center><img src="https://www.ucentral.edu.co/sites/default/files/inline-images/logosimbolo-ucentral-vigilada_4.png" width="250" align="up">

<H1 ALIGN=center><b>Maestría en Analítica de Datos<br>
<H3>Proyecto Métodos Estadísticos Para la Analítica de Datos<br>
<H4>Germán Andres Preciado Estupiñán
<H4>Cristian Fernando Rocha Vera
<H4>Angel Enrique Torres Benitez

---

# Violencia de género e intrafamiliar en Colombia



###Introducción

> Cuando hablamos de violencia intrafamiliar, debemos hacernos una invitación a comprender de que este término se refiera de manera directa a las acciones que puede tener una miembro o varios miembros contra los demás o algún miembro dentro de lo que se entiende como núcleo familiar, sin embargo al hablar de violencia intrafamiliar, es preciso mencionar que esta se puede presentar de varias maneras, sin ser la más común, es la que más se entiende como *agresión de tipo físico*, también se deben consideran las agresiones de *tipo verbal*, entendiéndose como la generación de actos violentos a través de la palabra y por otra parte también podemos tener la *violencia psicológica*, un poco menos mencionada, pero que presenta la misma importancia y relevancia que las demás, de manera que, cuando se habla de violencia intrafamiliar, este no solo se refiere a las agresiones físicas que un miembro pueda tener contra otra dentro de la familia, sino que también podemos encontrar las antes mencionadas.


<P ALIGN=center><img src="https://periodicoeldiario.com/wp-content/uploads/2021/05/802.jpg" width="300" >



### Problemática

> Los casos de violencia intrafamiliar de este estudio evidencian que las mujeres son el grupo más afectado por los diferentes tipos de violencia, a su vez, el grupo etéreo más afectado tanto en género masculino y femenino son la primera infancia, es decir, niños entre 0 y 6 años, de manera que, por medio de este estudio se pretende identificar de manera clara y con base en los datos los siguientes objetivos.

Objetivo:
+	Realizar una evaluación de los casos de violencia intrafamiliar con mayor incidencia por género y grupo de edad.

Objetivos Especificos:
+	Describir los casos de violencia intrafamiliar.
+	Analizar los casos de violencia intrafamiliar en la primera infancia.	
+	Caracterizar el tipo de agresor en los casos de violencia intrafamiliar.	



## Los Datos:

Los datos que se usaran para el presente estudio, se encuentran publicados como:

[04. Violencia de Género e intrafamiliar de enero 2015 a septiembre 2022 - Semana Epidemiológica 36](https://www.datos.gov.co/Salud-y-Protecci-n-Social/04-Violencia-de-G-nero-e-intrafamiliar-de-enero-20/sq8q-pnf5) 

en la pagina de: [Datos Abiertos](https://www.gov.co/)



###Cargue de librerias

In [ ]:
#Importamos las librerias necesarias
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import plotly.express as px
import seaborn as sns
%matplotlib inline 

In [ ]:
#Carga del archivo desde el drive de google
from google.colab import drive
drive.mount('/content/drive')

Mounted at /content/drive


###Dataframe de los casos de violencia:

In [ ]:
#Cargue del archivo csv y muestra de las primeras 5 filas
df = pd.read_csv("/content/drive/MyDrive/Estadistica/Bases/Violencia.csv")
df.head(4)

Orden Departamento    Municipio  semana   año Grupo edad Ciclo de vida  \
0      1    SANTANDER  BUCARAMANGA      23  2015    12 a 17  Adolescencia   
1      2    SANTANDER  BUCARAMANGA       5  2015    12 a 17  Adolescencia   
2      3    SANTANDER  BUCARAMANGA      29  2015    12 a 17  Adolescencia   
3      4    SANTANDER  BUCARAMANGA      16  2015    12 a 17  Adolescencia   

       sexo_               area_      Barrio  ... sust_vict   fec_hecho  \
0  Masculino  CABECERA MUNICIPAL  Transición  ...         2  12/06/2015   
1  Masculino  CABECERA MUNICIPAL     Betania  ...         2  04/02/2015   
2   Femenino  CABECERA MUNICIPAL  EL MIRADOR  ...         2  21/07/2015   
3   Femenino  CABECERA MUNICIPAL   María Paz  ...         2  22/04/2015   

    hora_hecho  escenario zona_conf  \
0  06:30:00 AM        1.0         2   
1  10:00:00 AM        1.0         2   
2  08:00:00 AM        2.0         2   
3  01:30:00 PM        1.0         2   

                                             nom_eve  \
0  VIGILANCIA EN SALUD PÚBLICA DE LAS VIOLENCIAS ...   
1  VIGILANCIA EN SALUD PÚBLICA DE LAS VIOLENCIAS ...   
2  VIGILANCIA EN SALUD PÚBLICA DE LAS VIOLENCIAS ...   
3  VIGILANCIA EN SALUD PÚBLICA DE LAS VIOLENCIAS ...   

                              nom_upgd  ndep_resi    nmun_resi          MES  
0           SERVICLINICOS DROMEDICA SA  SANTANDER  BUCARAMANGA    06. Junio  
1             HOSPITAL LOCAL DEL NORTE  SANTANDER  BUCARAMANGA  02. Febrero  
2  HOSPITAL UNIVERSITARIO DE SANTANDER  SANTANDER  BUCARAMANGA    07. Julio  
3             HOSPITAL LOCAL DEL NORTE  SANTANDER  BUCARAMANGA    04. Abril  

[4 rows x 32 columns]

###Comprobando los tipos de datos

In [ ]:
df.dtypes

Orden                         int64
Departamento                 object
Municipio                    object
semana                        int64
año                           int64
Grupo edad                   object
Ciclo de vida                object
sexo_                        object
area_                        object
Barrio                       object
Comuna                       object
Tipo de Seguridad Social     object
pac_hos_                      int64
con_fin_                      int64
version                      object
naturaleza                  float64
def_naturaleza               object
actividad                     int64
nom_actividad                object
edad_agre                   float64
sexo_agre                    object
parentezco_vict              object
sust_vict                     int64
fec_hecho                    object
hora_hecho                   object
escenario                   float64
zona_conf                     int64
nom_eve                     

###Columnas

In [ ]:
df.columns

Index(['Orden', 'Departamento', 'Municipio', 'semana', 'año', 'Grupo edad',
       'Ciclo de vida', 'sexo_', 'area_', 'Barrio', 'Comuna',
       'Tipo de Seguridad Social', 'pac_hos_', 'con_fin_', 'version',
       'naturaleza', 'def_naturaleza', 'actividad', 'nom_actividad',
       'edad_agre', 'sexo_agre', 'parentezco_vict', 'sust_vict', 'fec_hecho',
       'hora_hecho', 'escenario', 'zona_conf', 'nom_eve', 'nom_upgd',
       'ndep_resi', 'nmun_resi', 'MES'],
      dtype='object')

##Medidas descriptivas

In [ ]:
df.describe()

Orden        semana           año      pac_hos_      con_fin_  \
count  11157.000000  11157.000000  11157.000000  11157.000000  11157.000000   
mean    5579.000000     24.862597   2018.048400      1.713185      1.001524   
std     3220.892811     14.570800      2.360103      0.452295      0.047314   
min        1.000000      1.000000   2015.000000      1.000000      0.000000   
25%     2790.000000     12.000000   2016.000000      1.000000      1.000000   
50%     5579.000000     24.000000   2018.000000      2.000000      1.000000   
75%     8368.000000     37.000000   2020.000000      2.000000      1.000000   
max    11157.000000     53.000000   2022.000000      2.000000      2.000000   

         naturaleza     actividad    edad_agre     sust_vict     escenario  \
count  10724.000000  11157.000000  8605.000000  11157.000000  11154.000000   
mean       8.246363     27.073496    17.782336      1.984584      2.546441   
std       22.739312      3.382507    17.332278      0.123207      2.038640   
min        1.000000      8.000000     0.000000      1.000000      1.000000   
25%        1.000000     24.000000     0.000000      2.000000      2.000000   
50%        3.000000     26.000000    20.000000      2.000000      2.000000   
75%        3.000000     28.000000    30.000000      2.000000      2.000000   
max       99.000000     33.000000    99.000000      2.000000     12.000000   

          zona_conf  
count  11157.000000  
mean       1.998118  
std        0.043346  
min        1.000000  
25%        2.000000  
50%        2.000000  
75%        2.000000  
max        2.000000

In [ ]:
# Número total de filas y columnas
df.shape

(11157, 32)

In [ ]:
# Encontrar los valores nulos. 
print(df.isnull().sum())

Orden                          0
Departamento                   0
Municipio                      0
semana                         0
año                            0
Grupo edad                     0
Ciclo de vida                  0
sexo_                          0
area_                          0
Barrio                         0
Comuna                         0
Tipo de Seguridad Social       0
pac_hos_                       0
con_fin_                       0
version                        0
naturaleza                   433
def_naturaleza                 0
actividad                      0
nom_actividad                  0
edad_agre                   2552
sexo_agre                     28
parentezco_vict                0
sust_vict                      0
fec_hecho                      5
hora_hecho                  5492
escenario                      3
zona_conf                      0
nom_eve                        0
nom_upgd                       0
ndep_resi                      0
nmun_resi 

### Casos de Violencia por años

In [ ]:
#Agrupamos los casos de violencia por años
df.groupby('año').size()

año
2015    1839
2016    2141
2017    1407
2018    1099
2019    1220
2020    1077
2021    1124
2022    1250
dtype: int64

Analizamos los casos de violencia por años con una frecuencia relativa

In [ ]:
#Creamos una tabla que contenga la frecuencia relativa de los casos de violencia por años
tabla = df.groupby("año", as_index=False).size()
total = sum(tabla["size"])
#convertimos el año a string para poder ordenar el histrograma
tabla['año']= tabla['año'].apply(str)
tabla["Relativa"] = round(100*tabla["size"]/total,2)
tabla


año  size  Relativa
0  2015  1839     16.48
1  2016  2141     19.19
2  2017  1407     12.61
3  2018  1099      9.85
4  2019  1220     10.93
5  2020  1077      9.65
6  2021  1124     10.07
7  2022  1250     11.20

In [ ]:
tabla.dtypes

año          object
size          int64
Relativa    float64
dtype: object

### Histrograma de casos de violencia por años

In [ ]:
#Histograma casos por año
barras = px.bar(tabla, x="año", y='size', color='size', title="CASOS DE VIOLENCIA POR AÑOS",text='size', width=1000, height=500)
barras.update_xaxes(showgrid=True, ticks="outside")
barras.update_layout(xaxis={'categoryorder':'total descending'}) 

In [37]:
barras.show()

> Podemos identificar que el año 2016 fue el año donde se presento una mayor atención a casos de violencia con un total de 2.141 de casos reportados, seguido del 2015 con 1.839 casos.

### Casos de violencia por género

In [ ]:
df.groupby('sexo_').size()

sexo_
Femenino     7680
Masculino    3477
dtype: int64

In [ ]:
figura = px.pie(df, names="sexo_",title='Casos de Violencia por Genero', width=600, height=500, hole=0.3 ,color='sexo_', color_discrete_sequence=px.colors.sequential.GnBu_r, ) 
figura.update_traces(textposition='inside', textinfo='percent+label')
figura.show()

El género femenino es el mas afectado en la muestra con un 68% del total de  la poblacion.

### Tipos de Violencia

<!doctype html>
<html>
<head>
  <title>👩🏼👨🏼👧🏼👦🏽👶🏽</title>
  <meta charset=”utf-8”/>
</head>  
  <body>
  <h1 align="center"><b>Tipos de Violencia</b></h1>

**Violencia física:**

> Es toda acción que usa la fuerza física para causar dolor, malestar, lesiones, daños a la salud o al desarrollo del niño, niña o adolescente.

<!-- <center><img  src="https://wrmx00.epimg.net/radio/imagenes/2020/10/15/nacional/1602713391_985220_1602713808_noticia_normal.jpg" alt="Violencia física" width="200" height="100"  /></center>  -->

**Violencia sexual:**

> Se refiere a todo acto o comportamiento de tipo sexual ejercido sobre un niño, niña o adolescente, utilizando la fuerza o cualquier forma de coerción física, psicológica o emocional, aprovechando las condiciones de indefensión, de desigualdad y las relaciones de poder existentes entre víctima y agresor.

<!-- <center><img  src='https://www.elheraldo.co/sites/default/files/styles/414x310/public/articulo/2020/11/05/whatsapp_image_2020-11-05_at_7.31.22_pm.jpeg?itok=bhXddPpI' alt='violencia sexual' width="150" height="150"  /></center> -->

**Omisión o negligencia**

> Se refiere a la falta de protección y cuidado mínimo del niño, niña o adolescente por parte de los progenitores o encargados del cuidado. Existe negligencia cuando los responsables del cuidado no protegen de la exposición al peligro, ni atienden, o satisfacen las necesidades básicas de los niños, niñas y adolescentes, sean estas físicas, psicológicas, educativas o de salud, teniendo los medios, el conocimiento y acceso a la prestación de servicios.

<!-- <center><img  src='https://123laumunar.files.wordpress.com/2013/07/images-1.jpg' alt='violencia sexual' width="150" height="150"  /></center> -->

**Violencia psicológica:**

> Es toda acción u omisión destinada a degradar, discriminar o controlar las acciones, comportamientos, creencias y decisiones de los niños, niñas y adolescentes, a través de formas como:
Humillar
Rechazar
Aterrorizar
Aislar
Ser permisivos
Instrumentalizar
Restricción de la autonomía
Sobre exigencia

<!-- <center><img  src='https://archivo.amecopress.net/2020/png/foto_2_1_.png' alt='violencia sexual' width="150" height="150"  /></center> -->

  </body>
</html>













In [ ]:
#Diagrama de pastel que muestra los tipos de violencia
figura = px.pie(df, names="def_naturaleza",title='Tipos de Violencia', width=1100, height=800, hole=0.1 ,color='def_naturaleza', color_discrete_sequence=px.colors.sequential.Cividis_r ) 
figura.update_traces(textposition='inside', textinfo='percent+label')
figura.show()

In [ ]:
#Creamos una tabla que contenga la frecuencia relativa de los casos de violencia por años
tiposdeViolencia = df.groupby("def_naturaleza", as_index=False, dropna=False).size()
tiposdeViolencia.loc[tiposdeViolencia['def_naturaleza'] == 'Sin informacion', 'def_naturaleza'] = 'Sin Informacion'
total = sum(tiposdeViolencia["size"])
tiposdeViolencia["Relativa"] = round(100*tiposdeViolencia["size"]/total,2)
tiposdeViolencia

def_naturaleza  size  Relativa
0                                        Abuso sexual   952      8.53
1                                        Acoso sexual   141      1.26
2                            Actos sexuales violencia   118      1.06
3   Explotacion sexual, comercial niños, adolescentes     3      0.03
4                              Negligencia y abandono  4480     40.15
5                                Otros actos sexuales    45      0.40
6                                     Sin Informacion   433      3.88
7                                     Sin Informacion   630      5.65
8                                           Violacion   239      2.14
9                                    Violencia fisica  3622     32.46
10                              Violencia psicologica   494      4.43

In [ ]:
#Histograma de tipos de violencia
barras = px.bar(tiposdeViolencia, x="def_naturaleza", y='Relativa', color='Relativa', title="TIPOS DE VIOLENCIA",text='Relativa', width=1300, height=800)
barras.update_xaxes(showgrid=True, ticks="outside")
barras.update_layout(xaxis={'categoryorder':'total descending'}) 
barras.show()

### Casos por grupo etario

* Primera infancia (de 0 a 6 años)
* Infancia (de 7 a 11 años)
* Adolescencia (de 12 a 17 años)
* Jovenes (de 18 a 28 años)
* Adultez (de 29 a 59 años)
* Vejez (de 60 años en adelante)


In [ ]:
tabla1 = df.groupby('Ciclo de vida', dropna=False, as_index=False).size()
total1 = sum(tabla1["size"])
tabla1["Relativa"] = round(100*tabla1["size"]/total1 ,2)
tabla1

Ciclo de vida  size  Relativa
0      Adolescencia  2034     18.23
1           Adultez  1849     16.57
2       ERROR: #N/A     1      0.01
3          Infancia  1440     12.91
4           Jovenes  2079     18.63
5     Persona Mayor   318      2.85
6  Primera infancia  3436     30.80

#### Participacion porcentual por grupo etario

In [ ]:
barras1 = px.bar(tabla1, x='Ciclo de vida', y="Relativa", color='Relativa' , title="Casos por grupo etereo",text_auto=True,width=1000, height=600)
barras1.update_layout(xaxis={'categoryorder':'total descending'}) 
barras1.update_xaxes(showgrid=True, ticks="outside")
barras1.show()

La primera infancia es la más afectada con una participacion del 30%, en segundo lugar están los jovenes con un 18.63% y en tercer lugar tenemos a la adolescencia con 18.23%.  mirar stact

In [ ]:
variables = ["Grupo edad",'sexo_']
datosA = df[variables]
datosA = datosA.dropna()
fig = px.box(datosA, y ="Grupo edad", color='sexo_')
fig.show()

En los hombres el grupo etario mas afectado está entre los 0 y 11 años, en las mujeres el rango es mas amplio va desde los 0 a los 28 años.

### Analisis de datos en las Mujeres

In [ ]:
datosMujeres = df[df["sexo_"]=="Femenino"]

Analisis por tipo de agresión

In [ ]:
tabla2 = datosMujeres.groupby("def_naturaleza", as_index=False).size()
total2 = sum(tabla2["size"])
tabla2["Relativa"] = 100*tabla2["size"]/total2
tabla2

def_naturaleza  size   Relativa
0                                        Abuso sexual   833  10.846354
1                                        Acoso sexual   120   1.562500
2                            Actos sexuales violencia    92   1.197917
3   Explotacion sexual, comercial niños, adolescentes     2   0.026042
4                              Negligencia y abandono  2294  29.869792
5                                Otros actos sexuales    36   0.468750
6                                     Sin Informacion   351   4.570312
7                                     Sin informacion   533   6.940104
8                                           Violacion   215   2.799479
9                                    Violencia fisica  2795  36.393229
10                              Violencia psicologica   409   5.325521

In [ ]:
barras2 = px.bar(tabla2, x="def_naturaleza", y="Relativa", color='Relativa' ,title="Violencia en las mujeres",text_auto=True,width=1200, height=700)
barras2.show()

En las mujeres la violencia fisica, la neglicencia y abandono y el abuso sexual son los tipos de violencia más encontrados en esta muestra con un total del 75% de los demas casos. 

### Analisis de datos en los hombres

In [ ]:
datosHombres = df[df["sexo_"]=="Masculino"]

In [ ]:
tabla3 = datosHombres.groupby("def_naturaleza", as_index=False).size()
total3 = sum(tabla3["size"])
tabla3["Relativa"] = 100*tabla3["size"]/total3
tabla3

def_naturaleza  size   Relativa
0                                        Abuso sexual   119   3.422491
1                                        Acoso sexual    21   0.603969
2                            Actos sexuales violencia    26   0.747771
3   Explotacion sexual, comercial niños, adolescentes     1   0.028760
4                              Negligencia y abandono  2186  62.870290
5                                Otros actos sexuales     9   0.258844
6                                     Sin Informacion    82   2.358355
7                                     Sin informacion    97   2.789761
8                                           Violacion    24   0.690250
9                                    Violencia fisica   827  23.784872
10                              Violencia psicologica    85   2.444636

In [ ]:
barras3 = px.bar(tabla3, x="def_naturaleza", y="Relativa", color='Relativa' ,title="Violencia en los hombres",text_auto=True,width=1200, height=700)
barras3.show()

En los hombres la negligencia y abandono es la cifra mas representativa en la muestra con una participacion del 62% seguido de Violencia fisica con 23%

# Analisis de la Violencia por Negligencia y Abandono

**Maltrato por negligencia:**

Consiste en la falta de satisfacción de las necesidades básicas del niño y del adolescente, como es la comida, ropa, albergue, atención o supervisión necesaria para el desarrollo de su personalidad y el crecimiento óptimo, evaluando las posibilidades socio económicas de los adultos responsables.

In [ ]:
Negligencia = df.loc[:, 'def_naturaleza'] == 'Negligencia y abandono'
df_Negligencia = df.loc[Negligencia]
df_Negligencia.head()

Orden Departamento    Municipio  semana   año Grupo edad Ciclo de vida  \
0       1    SANTANDER  BUCARAMANGA      23  2015    12 a 17  Adolescencia   
2       3    SANTANDER  BUCARAMANGA      29  2015    12 a 17  Adolescencia   
7       8    SANTANDER  BUCARAMANGA      45  2015    12 a 17  Adolescencia   
8       9    SANTANDER  BUCARAMANGA      26  2015    7  a 11      Infancia   
14     15    SANTANDER  BUCARAMANGA      13  2015    12 a 17  Adolescencia   

        sexo_               area_            Barrio  ... sust_vict  \
0   Masculino  CABECERA MUNICIPAL        Transición  ...         2   
2    Femenino  CABECERA MUNICIPAL        EL MIRADOR  ...         2   
7    Femenino  CABECERA MUNICIPAL  Colseguros Norte  ...         2   
8    Femenino  CABECERA MUNICIPAL         María Paz  ...         2   
14   Femenino  CABECERA MUNICIPAL         Colorados  ...         2   

     fec_hecho   hora_hecho  escenario zona_conf  \
0   12/06/2015  06:30:00 AM        1.0         2   
2   21/07/2015  08:00:00 AM        2.0         2   
7   14/11/2015  08:00:00 PM        1.0         2   
8   01/07/2015  04:00:00 PM        2.0         2   
14  01/04/2015  03:00:00 PM        1.0         2   

                                              nom_eve  \
0   VIGILANCIA EN SALUD PÚBLICA DE LAS VIOLENCIAS ...   
2   VIGILANCIA EN SALUD PÚBLICA DE LAS VIOLENCIAS ...   
7   VIGILANCIA EN SALUD PÚBLICA DE LAS VIOLENCIAS ...   
8   VIGILANCIA EN SALUD PÚBLICA DE LAS VIOLENCIAS ...   
14  VIGILANCIA EN SALUD PÚBLICA DE LAS VIOLENCIAS ...   

                               nom_upgd  ndep_resi    nmun_resi            MES  
0            SERVICLINICOS DROMEDICA SA  SANTANDER  BUCARAMANGA      06. Junio  
2   HOSPITAL UNIVERSITARIO DE SANTANDER  SANTANDER  BUCARAMANGA      07. Julio  
7            SERVICLINICOS DROMEDICA SA  SANTANDER  BUCARAMANGA  11. Noviembre  
8              HOSPITAL LOCAL DEL NORTE  SANTANDER  BUCARAMANGA      07. Julio  
14             HOSPITAL LOCAL DEL NORTE  SANTANDER  BUCARAMANGA      04. Abril  

[5 rows x 32 columns]

In [ ]:
tabla4 = df_Negligencia.groupby("Ciclo de vida", as_index=False).size()
total4 = sum(tabla4["size"])
tabla4["Relativa"] = 100*tabla4["size"]/total4
tabla4

Ciclo de vida  size   Relativa
0      Adolescencia   530  11.830357
1           Adultez   111   2.477679
2       ERROR: #N/A     1   0.022321
3          Infancia   760  16.964286
4           Jovenes   338   7.544643
5     Persona Mayor   149   3.325893
6  Primera infancia  2591  57.834821

In [ ]:
barras4 = px.bar(tabla4, x="Ciclo de vida", y="Relativa", color='size' ,title="Negligencia y abandono por edades",text_auto=True,width=1000, height=500)
barras4.show()

In [ ]:
tabla5 = df_Negligencia.groupby('sexo_', as_index=False).size()
total5 = sum(tabla5["size"])
tabla5["Relativa"] = 100*tabla5["size"]/total5
tabla5

sexo_  size   Relativa
0   Femenino  2294  51.205357
1  Masculino  2186  48.794643

In [ ]:
df_Negligencia.iloc[:, [0,6,7]]  # Primera, tercera y segunda columnas

Orden     Ciclo de vida      sexo_
0          1      Adolescencia  Masculino
2          3      Adolescencia   Femenino
7          8      Adolescencia   Femenino
8          9          Infancia   Femenino
14        15      Adolescencia   Femenino
...      ...               ...        ...
11143  11144          Infancia  Masculino
11146  11147  Primera infancia   Femenino
11149  11150  Primera infancia   Femenino
11152  11153          Infancia  Masculino
11156  11157          Infancia   Femenino

[4480 rows x 3 columns]

In [ ]:
figura2 = px.pie(df_Negligencia, names="Ciclo de vida",title='Negligencia y Abandono por Edad', width=400, height=400) 
figura2.show()


In [ ]:
figura3 = px.pie(df_Negligencia, names="sexo_",title='Negligencia y Abandono por genero', width=400, height=400) 
figura3.show()